app/assets/translations/appConstants/bebbo/MKmk.ts

```diff
@@ -314,7 +315,7 @@ const MKmk = {
                 developScreenCompletechartText: "Честитки, вашето дете ги достигна сите развојни одредници!",
                 noDataTxt: "Не се достапни податоци",
                 hcEditHeaderTitle: "Променете ги податоците за прегледот кај лекар",
-                genderLabel: "Детето е:",
+                genderLabel: "Детето е",
```

Comment:
Should we change "Search in Bebbo" to "Search" in this file?

Reply:
Please ignore 'search' string for now as we are not using it.